frontend/src/components/rule-edit/generation-rule/GenerationRuleList.vue

```diff
@@ -54,36 +54,35 @@ const editedRule = computed(() =>
 </script>
 
 <template>
-  <CCard class="border bg-light p-2">
-    <CCardBody>
-      <div v-if="generation_rules.length === 0">
-        <h4 class="text-center text-muted my-4">No Destinations Defined</h4>
-      </div>
-      <FormKit
-        v-else
-        type="list"
-        name="generation_rules"
-        v-model="generation_rules"
+  <CListGroup>
+    <FormKit
+      v-if="generation_rules.length > 0"
+      type="list"
+      name="generation_rules"
+      v-model="generation_rules"
+    >
+      <template v-for="(gr, index) in generation_rules" :key="gr.id">
+        <GenerationRuleListItem
+          :rule="gr"
+          @edit="() => handleButtonClicked(index)"
+          @delete="() => handleRuleDeleted(index)"
+        />
+      </template>
+    </FormKit>
+    <CListGroupItem class="bg-light">
+      <h4 v-if="generation_rules.length === 0" class="text-center text-muted my-4">No Destinations Defined</h4>
+      <div class="text-center">
+      <CButton
+        @click.prevent="() => handleButtonClicked(-1)"
+        color="primary"
```

Comment:
I had set the color to be secondary when there's already a rule because I thought it's a guideline to have only one primary button on the page (to make it clear what the next step is), and the Create Rule button is already primary. Do you think it's OK to have 2 primary buttons? I'll totally defer to your opinion on the matter, I just wanted to point this out.

Reply:
+1 will add it back